cmake/BipedalLocomotionFrameworkFindDependencies.cmake

```diff
@@ -131,6 +131,9 @@ checkandset_dependency(Eigen3)
 find_package(Qhull 8.0.0 QUIET)
 checkandset_dependency(Qhull)
 
+find_package(Casadi QUIET)
+checkandset_dependency(Casadi)
```

Comment:
I would suggest using ``casadi`` lowercase since it is lowercase when you clone the repo and its config files go into ``lib/cmake/casadi`` (lowercase).

By the way, I start hating the ``cmake`` machinery for the dependencies. It does not provide any useful information of why the dependency has not been found.

Reply:
> By the way, I start hating the cmake machinery for the dependencies. It does not provide any useful information of why the dependency has not been found.

:heart:

Reply:
Not sure which machinery you refer, if the `checkandsetdeps` functions or actually `find_package`, but for `find_package` on recent CMake versions  (>= 3.17) there is the convenient CMake variable [`CMAKE_FIND_DEBUG_MODE`](https://cmake.org/cmake/help/v3.17/variable/CMAKE_FIND_DEBUG_MODE.html). You can also  invoke cmake with the ` --debug-find` flag.

Reply:
No no, the output of ``find_package`` is pretty clear to me. I was referring to https://github.com/dic-iit/bipedal-locomotion-framework/blob/master/cmake/BipedalLocomotionFrameworkFindDependencies.cmake#L95-L112, but then I realized that those lines were the consequence, not the cause. Then I opened #66

Reply:
Done in ec2c99c6384f330997b65a7b8c6308b50b1f49fb